README.rst

```diff
@@ -63,8 +63,8 @@ Features
 Requirements
 --------------
 
-* Django 1.8, 1.10, 1.11, or 2.0
-* Python 2.7, 3.4, 3.5, or 3.6
+* Django 1.11 or 2.0
+* Python 3.4, 3.5, or 3.6
```

Comment:
Should these be updated in line with setup.py? ie
```
* Django 1.11, 2.1, 2.2, or 3.0
* Python 3.6, 3.7, or 3.8
```

Reply:
Great catch @MattFisher.

I'll update it, thank you.

Reply:
Thanks! Looking forward to this patch landing because we want to move to Django 3.0.